tests/unit/freezable-test.js

```diff
@@ -0,0 +1,53 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+
+module('Ember.Freezable');
+
+const { get, set } = Ember;
+
+test('frozenCopy should be deprecated', assert => {
+  assert.expectDeprecation(() => {
+    let Obj = Ember.Object.extend(Ember.Freezable, Ember.Copyable, {
+      copy() {
+        return Obj.create();
+      }
+    });
+
+    Obj.create().frozenCopy();
+  }, '`frozenCopy` is deprecated, use `Object.freeze` instead.');
+});
+
+test('Ember.Freezable should be deprecated', assert => {
```

Comment:
can you normalize to `function(assert) {`?

Reply:
@locks. Fixed!